adapters/minutemedia/minutemedia.go

```diff
@@ -102,6 +102,7 @@ func extractOrg(openRTBRequest *openrtb2.BidRequest) (string, error) {
 			return "", fmt.Errorf("failed to unmarshal ImpExtMinuteMedia: %w", err)
 		}
 
+		//nolint: staticcheck // SA4004: the surrounding loop is unconditionally terminated
 		return strings.TrimSpace(impExt.Org), nil
```

Comment:
Same here, I believe we can refactor to avoid the `nolint` comment:
```
 92   func extractOrg(openRTBRequest *openrtb2.BidRequest) (string, error) {
 93       var err error
 94 -     for _, imp := range openRTBRequest.Imp {
    +     if len(openRTBRequest.Imp) == 0 {
    +         return "", errors.New("no imps in bid request")
    +     }
 95           var bidderExt adapters.ExtImpBidder
 96           if err = json.Unmarshal(imp.Ext, &bidderExt); err != nil {
 97               return "", fmt.Errorf("failed to unmarshal bidderExt: %w", err)
 98           }
 99  
100           var impExt openrtb_ext.ImpExtMinuteMedia
101           if err = json.Unmarshal(bidderExt.Bidder, &impExt); err != nil {
102               return "", fmt.Errorf("failed to unmarshal ImpExtMinuteMedia: %w", err)
103           }
104  
105 -         //nolint: staticcheck // SA4004: the surrounding loop is unconditionally terminated
106 -         return strings.TrimSpace(impExt.Org), nil
107 -     }
108 -
109 -     return "", errors.New("no imps in bid request")
    +     return strings.TrimSpace(impExt.Org), nil
110   }
adapters/minutemedia/minutemedia.go [RO]
```

Reply:
done in 6d4f03aa

Reply:
Thank you